api/nodecore/v1alpha1/common.go

```diff
@@ -6,14 +6,14 @@ const (
 	PhaseFailed  Phase = "Failed"
 	PhaseRunning Phase = "Running"
 	PhaseIdle    Phase = "Idle"
-	PhaseTimeout Phase = "Timeout"
+	PhaseTimeout Phase = "Timed Out"
 )
 
 // GenericRef represents a reference to a generic Kubernetes resource,
 // and it is composed of the resource name and (optionally) its namespace.
 type GenericRef struct {
 	// The name of the resource to be referenced.
-	Name string `json:"name"`
+	Name string `json:"name,omitempty"`
```

Comment:
```suggestion
	Name string `json:"name"`
```

Reply:
Already fixed, it was an old conversation (see change in `api/nodecore/v1alpha1/common.go`)

Reply:
Now done done doneee